extensions/blocks/donations/editor.scss

```diff
@@ -0,0 +1,72 @@
+/**
+ * Editor styles for Donations
+ */
+.wp-block-jetpack-donations .donations__container {
```

Comment:
I'd recommend switching from spaces to tabs, to ensure that the next person editing this file does not fix all that at once.